test/executor/delete_test.cpp

```diff
@@ -47,8 +47,16 @@ void ShowTable(std::string database_name, std::string table_name) {
   executor::ExecuteResult status;
   std::vector<type::Value> params;
   std::vector<StatementResult> result;
+
   optimizer::Optimizer optimizer;
+
+//  optimizer::SimpleOptimizer optimizer;
```

Comment:
Why is this line still here? That was deleted in the latest version of the master branch.